### DISCUSSION TITLE
Exploring vector to expose metrics related logs to prometheus

### DISCUSSION THREAD

**Discussion by samuelarogbonlo** [vectordotdev/vector#7778] [Q&A]
### Current Vector Version
```
v0.14.0
```
### Use-cases
I have some logs from Cloudwtach ECS services, some others from web servers and even K8s pods. i want to use Vector to explore metrics related logs from maybe the webserver to Prometheus. 

### Attempted Solutions

I have gone through all the documentation but it seems nothing like that exists but I am of a surety that it could be possible with Vector

### Proposal

I am proposing that some sort of solution is rolled out here and probably merged with the repository then finally added to the documentation. But if there is already an existing solution, I'd like to know about it - it'd really be helpful in my project. 

I hope someone could help me beat this issue. thanks 

@blt @jamtur01 @hdhoang @neoeinstein @thoughtpolice and any others that could help

**Comment by JeanMertz** [comment]
Given the age of the discussion, I'm uncertain if this is still something you are working on, but before I can try and help you, I would need a bit more details on this:

> explore metrics related logs from maybe the webserver to Prometheus

Could you tell me what you mean by "metrics related logs". Do you have any examples on what exactly you are trying to do with Vector?

Vector is great for getting logs from the sources you mentioned, and with the `remap` and `metric_to_log` transform you can explore any metrics as you see fit.
